ext/php7/compatibility.h

```diff
@@ -54,37 +45,18 @@
 #define _GET_UNUSED_MACRO_OF_ARITY(_1, _2, _3, _4, _5, ARITY, ...) UNUSED_##ARITY
 #define UNUSED(...) _GET_UNUSED_MACRO_OF_ARITY(__VA_ARGS__, 5, 4, 3, 2, 1)(__VA_ARGS__)
 
-#if PHP_VERSION_ID < 70000
-#define PHP5_UNUSED(...) UNUSED(__VA_ARGS__)
-#define PHP7_UNUSED(...) /* unused unused */
-#else
 #define PHP5_UNUSED(...) /* unused unused */
 #define PHP7_UNUSED(...) UNUSED(__VA_ARGS__)
```

Comment:
In a follow up PR we can:

1) Replace all `PHP7_UNUSED` with `UNUSED`
2) Remove references to `PHP5_UNUSED`
3) Remove the `PHP*_UNUSED` macros

Reply:
Good points @SammyK! Since I am not ready to push the branch that is supposed to start from this, I decided to do it so we a re a step closer to the final goal:
1. https://github.com/DataDog/dd-trace-php/pull/1162/commits/83675d6e1d671ddb50c98823a8fb1922dd2f9b75
2. 3. https://github.com/DataDog/dd-trace-php/pull/1162/commits/607f538593914a7c43d4fc22585a057c5a89f94d

Also, I had forgotten about https://github.com/DataDog/dd-trace-php/pull/1162/commits/bf45c881432e82311a3bbb96e472e4f6a785c13e